ci/infra/vmware/main.tf

```diff
@@ -0,0 +1,41 @@
+provider "vsphere" {
```

Comment:
I just realized that the version requires > 1.17.
At OBS, we provide version 1.16.1
At IBS, we provide version 1.11.0
Do we have to update the package?

cc @cclhsu

Reply:
Good catch! @c3y1huang @cclhsu ?

Reply:
Yes it’s required by hardware_version.

Reply:
@cclhsu as @c3y1huang said we need to upgrade the vsphere terraform plugin to the version. Let's follow up on another ticket. Thanks.